src/pybind/mgr/tox.ini

```diff
@@ -65,12 +65,11 @@ commands =
     pytest {posargs:cephadm/tests/test_ssh.py}
 
 
-[testenv:mypy]
+[testenv:{,py37-,py38-,py39-,py310-}mypy]
```

Comment:
to be clear, this gives us the ability to run something like `tox -e py37-mypy`, but by default we'll still just run `mypy`?

Reply:
Yep. tox supports expansions like those of bash. So `{,py37-,py38-,py39-,py310-}mypy` expands to mypy, py37-mypy, py39-mypy, py310-mypy.  I got lazy and didn't figure we'd need anything newer than 3.10 for now.